{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-20396",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/github.com\/CESNET\/libyang\/commit\/a1f17693904ed6fecc8902c747fc50a8f20e6af8",
                    "name": "https:\/\/github.com\/CESNET\/libyang\/commit\/a1f17693904ed6fecc8902c747fc50a8f20e6af8",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/github.com\/CESNET\/libyang\/compare\/v0.16-r3...v1.0-r1",
                    "name": "https:\/\/github.com\/CESNET\/libyang\/compare\/v0.16-r3...v1.0-r1",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/github.com\/CESNET\/libyang\/issues\/740",
                    "name": "https:\/\/github.com\/CESNET\/libyang\/issues\/740",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "A segmentation fault is present in yyparse in libyang before v1.0-r1 due to a malformed pattern statement value during lys_parse_path parsing."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2020-01-22T22:15Z",
    "lastModifiedDate": "2020-01-22T22:15Z"
}